packages/block-library/src/latest-posts/icon.js

```diff
@@ -1,17 +1,10 @@
 /**
  * WordPress dependencies
  */
-import { Path, Rect, SVG } from '@wordpress/components';
+import { Path, SVG } from '@wordpress/components';
 
 export default (
 	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-		<Path d="M0,0h24v24H0V0z" fill="none" />
-		<Rect x="11" y="7" width="6" height="2" />
-		<Rect x="11" y="11" width="6" height="2" />
-		<Rect x="11" y="15" width="6" height="2" />
-		<Rect x="7" y="7" width="2" height="2" />
-		<Rect x="7" y="11" width="2" height="2" />
-		<Rect x="7" y="15" width="2" height="2" />
-		<Path d="M20.1,3H3.9C3.4,3,3,3.4,3,3.9v16.2C3,20.5,3.4,21,3.9,21h16.2c0.4,0,0.9-0.5,0.9-0.9V3.9C21,3.4,20.5,3,20.1,3z M19,19H5V5h14V19z" />
+		<Path d="M18 4H6c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm.5 14c0 .3-.2.5-.5.5H6c-.3 0-.5-.2-.5-.5V6c0-.3.2-.5.5-.5h12c.3 0 .5.2.5.5v12zM7 11h2V9H7v2zm0 4h2v-2H7v2zm3-4h7V9h-7v2zm0 4h7v-2h-7v2z" />
```

Comment:
Same, I thought I already moved this to the icons package? I believe this file is not used at all and should be removed.